material/src/main/java/org/odk/collect/material/MaterialProgressDialogFragment.java

```diff
@@ -52,29 +62,18 @@ public void setTitle(String title) {
     }
 
     public void setMessage(String message) {
-        setArgument(message, MESSAGE);
+        setArgument(MESSAGE, message);
 
         AlertDialog dialog = (AlertDialog) getDialog();
         if (dialog != null) {
             setupView(dialog);
         }
     }
 
-    /*
```

Comment:
Are you sure it's not needed anymore? I remember that long time ago it was something I was able to reproduce or at least I saw it in crash reports.

Reply:
Ah sorry I thought I left a comment about this : `DialogFragmentUtils#showIfNotShowing` has the same workaround, so it was just duplicated rather than not needed.